tests/unit/inc/health-check-test.php

```diff
@@ -131,17 +128,20 @@ public function test_get_test_result() {
 			],
 			$this->instance->get_test_result()
 		);
+		$this->assertTrue( $this->instance->has_run );
 	}
 
 	/**
 	 * Tests the get_async_test_result method.
 	 *
-	 * @covers WPSEO_Health_Check::get_async_test_result
+	 * @covers WPSEO_Health_Check::get_JSON_test_result
 	 */
-	public function get_async_test_result() {
-		$this->instance
-			->shouldReceive( 'run' )
-			->once();
+	public function test_get_async_test_result() {
```

Comment:
```suggestion
public function test_get_json_test_result() {
```

(letting the test name reflect the name of the method under test)

Reply:
Actually - scratch the above! The test name is correct, the `@covers` looks to be wrong.